### DISCUSSION TITLE
Newline

### DISCUSSION THREAD

**Discussion by tgaldi** [inkle/ink#696] [Writing in ink]
Hi,

Maybe this is a silly question, but how do you use newlines in ink text. I see that I can add `<br>`, but I would prefer Ink to just respect newlines in the text itself, i.e:

```
Line 1

Line 2
```

Thanks!

**Comment by Selsynn** [comment]
It's an html problem. 
If you want to have the exact same output you can have it also surrounded by a `<pre>` `</pre>` tag.
Like for example:
```
<pre>
Once
upon a time...


 * There were two choices.
 * There were four lines of content.
    A bit of text

- They 
lived 
happily 
ever 
after.
    -> END
</pre>
```
Note that it may not work in the web export and will definitively not work in the unity export

**Reply by tgaldi** [reply]
Ah, ok thank you. I'm using Unity so I guess I'll need to add `<br>`

**Reply by Selsynn** [reply]
If you are using unity, you can also just modify the code that is handling it. 
the easy solution is to add a custom string/character (usually I would use `\n` or `<br/>` and to parse it if that's not already parsed into a new line. I believe both may work directly in a new line

**Reply by tgaldi** [reply]
Yeah I'm using TextMeshPro so it already handles `<br>` and `\n` so I won't need to do any custom parsing, I was just curious if there's a ink compiler setting I could use to have ink parse the newlines so that the text would be WYSIWYG.
